addons/project_sale_expense/tests/test_reinvoice_expense_analytic_distribution.py

```diff
@@ -0,0 +1,55 @@
+from odoo.tests.common import tagged
+from odoo.addons.sale.tests.common import TestSaleCommonBase
+
+
+@tagged('-at_install', 'post_install')
+class TestExpenseReinvoiceAnalyticDistribution(TestSaleCommonBase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+    def test_expense_reinvoice_analytic_distribution(self):
+        sale_order = self.env['sale.order'].with_context(mail_notrack=True, mail_create_nolog=True).create({
+            'partner_id': self.partner_a.id,
+            'partner_invoice_id': self.partner_a.id,
+            'partner_shipping_id': self.partner_a.id,
+            'analytic_account_id': self.analytic_account.id,
+            'pricelist_id': self.company_data['default_pricelist'].id,
+            'order_line': [
+                (0, 0, {
+                    'name': self.company_data['product_order_cost'].name,
```

Comment:
```suggestion
            'order_line': [
                Command.create({
                    'name': self.company_data['product_order_cost'].name,
```
We have helpers for that, please never use tuple commands without the helpers as it's less clear